core/server/worker/src/main/java/alluxio/worker/block/CacheRequestManager.java

```diff
@@ -257,11 +298,92 @@ public void run() {
         } else {
           ASYNC_CACHE_FAILED_BLOCKS.inc();
         }
-        mPendingRequests.remove(blockId);
+        mPendingAsyncRequests.remove(blockId);
       }
     }
   }
+  /**
+   * CacheTask is a callable task that can be considered equal if
+   * the blockId of the request is the same.
+   */
+  @VisibleForTesting
+  class CacheTask implements Callable<Boolean> {
```

Comment:
This seems like a duplicate of AsyncCacheTask. I have the same comment below.
BlockWorker already provides a distinction of async/sync. I don't see why we need to maintain two different names for the same storage format.